PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,10 @@
+## Pull Request Guidelines for Authors
+
+Summary of [RFC66 - Pull Request & Code Review Standards](https://safetyculture.atlassian.net/wiki/spaces/ENG/pages/2881716914/RFC66+Pull+Request+Code+Review+Standards):
+
+<!-- Go over all the following points, and put an `x` in the boxes that are true -->
+- [ ] Have you included a short summary of what the PR does, a description of the problem being solved and the pros/cons of the approach?
```

Comment:
Rather than a checkbox, these could be headings in the template. That would implicitly encourage authors to fill out that information.

Reply:
Switched to comments in [ac6a119](https://github.com/SafetyCulture/.github/pull/3/commits/ac6a11977fe104f4fa661025f4a88fe9ecb385f4)